### PR TITLE
action: allow configurable policy in AutoPlace action

### PR DESCRIPTION
**Pull request by ahesford**
Closes: #1784.

### CODE REVIEW

**Comment by stefonarch** [comment]
Nice, thanks!

**Comment by tsujan** [comment]
Sorry if I clutter this page by adding a user comment, but I just wanted to tell why this patch is good for Qt users: They've got used to see *parentless* dialogs centered on the screen (Qt does it under X11), and with this change, they could enforce a central position for some of them while using the automatic policy (= smart positioning) for other windows. I made a patch for my personal use to do so, but this PR seems much better.

**Comment by johanmalm** [comment]
> @johanmalm @Consolatis @jlindgren90 it would be nice to merge this for `0.7.2`

No. The release is tomorrow and we only do bug-fixes in the cool down period.

At the start of the cool-down period, we might make an exception for a low-risk feature - but not the day before. We value stability too much.

**Comment by 01micko** [comment]
Fair enough

**Comment by tokyo4j** [comment]
If we merge this, I think `MoveToCursor` should be deprecated.

File: src/action.c [hunk 1]
```diff
@@ -415,6 +415,12 @@ action_arg_from_xml_node(struct action *action, const char *nodename, const char
 			goto cleanup;
 		}
 		break;
+	case ACTION_TYPE_AUTO_PLACE:
+		if (!strcmp(argument, "policy")) {
+			action_arg_add_str(action, argument, content);
```
[the remark this answers is not in the document]
Comment (ahesford) [reply src/action.c]: Done

**Comment by johanmalm** [comment]
LGTM

**Pull request merged**